pkg/resolver/searchComplete.go

```diff
@@ -122,30 +136,32 @@ func (s *SearchCompleteResult) searchCompleteResults(ctx context.Context) ([]*st
 			srchCompleteOut = append(srchCompleteOut, &prop)
 		}
 	}
-	isNumber := isNumber(srchCompleteOut)
-	if isNumber { //check if valid number
-		isNumber := "isNumber"
-		srchCompleteOutNum := []*string{&isNumber} //isNumber should be the first argument if the property is a number
-		// Sort the values in srchCompleteOut
-		sort.Slice(srchCompleteOut, func(i, j int) bool {
-			numA, _ := strconv.Atoi(*srchCompleteOut[i])
-			numB, _ := strconv.Atoi(*srchCompleteOut[j])
-			return numA < numB
-		})
-		if len(srchCompleteOut) > 1 {
-			// Pass only the min and max values of the numbers to show the range in the UI
-			srchCompleteOut = append(srchCompleteOutNum, srchCompleteOut[0], srchCompleteOut[len(srchCompleteOut)-1])
-		} else {
-			srchCompleteOut = append(srchCompleteOutNum, srchCompleteOut...)
-		}
+	if len(srchCompleteOut) > 0 {
+		//Check if results are date or number
+		isNumber := isNumber(srchCompleteOut)
+		if isNumber { //check if valid number
+			isNumber := "isNumber"
+			srchCompleteOutNum := []*string{&isNumber} //isNumber should be the first argument if the property is a number
```

Comment:
I got confused by the scope of `isNumber`, could you use different variable names?
```suggestion
		isNumber := isNumber(srchCompleteOut)
		if isNumber { //check if valid number
			isNumberStr := "isNumber"
			srchCompleteOutNum := []*string{&isNumberStr} //isNumber should be the first argument if the property is a number
```